exported-schema.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "7.2.0",
+  "version": "7.3.0",
```

Comment:
for reference, we increment at release. you can do whatever while you work, but make sure you don't merge manual version changes (which it looks like you've since reverted, anyway). 

FWIW, you should be able to use `npm link` to test without changing the version.